src/App.css

```diff
@@ -1,39 +1,17 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+* {
+  margin: 0;
+  padding: 0;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+body {
+  background-color: rgb(105, 94, 94);
```

Comment:
- [ ] Kindy  note that the design guidelines need to be followed, make sure your design look the same like the template, pick one color, use the same layout, fontsize, margin, position ....Below you will see the a video where I point more about the design guideline

https://drive.google.com/file/d/1GcEqvu-m97SMtc5Qfsa2S05lcLeks1fV/view?usp=sharing


Your Design | The template 
-----|-----
![image](https://github.com/melkamu12/react-capstone/assets/65068771/eb6f85c2-141f-4a87-9c1d-4369e676b02b) ![image](https://github.com/melkamu12/react-capstone/assets/65068771/c655e14e-67f5-4e71-92c3-df6dc20ccd3a) | ![image](https://github.com/melkamu12/react-capstone/assets/65068771/78d9203e-7673-477e-8bc5-cbc043dea4df) ![image](https://github.com/melkamu12/react-capstone/assets/65068771/ec0e12c5-60fa-4e7d-a756-b1d3bf599491) ![image](https://github.com/melkamu12/react-capstone/assets/65068771/f3984535-7c1f-4ed7-81a5-76030f9d0a3d)